src/styles/product-card.scss

```diff
@@ -0,0 +1,98 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-size: inherit;
+    line-height: 18px;
+    height: 36px;
+    color: $main-color;
+    font-weight: 500;
+    margin-top: 40px;
+    margin-bottom: 5px;
+    width: 100%;
+  }
+
+  &__code {
+    display: flex;
+    margin: 0;
+    justify-content: flex-start;
+    width: 100%;
+    color: $secondary-color;
+    font-size: 10px;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 18px;
+    margin-bottom: 7px;
+
+    &-text {
+      display: flex;
+      align-items: center;
+      margin: 0;
+      font-size: 10px;
+      height: 16px;
+      font-weight: 400;
+      color: $main-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    font-size: 12px;
+
+    &-text {
+      color: $secondary-color;
+      font-weight: 400;
+    }
+
+    &-price {
+      color: $main-color;
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
+    background-color: $button-color;
+    border-radius: 5px;
+    border: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: white;
+    text-decoration: none;
+
+    &:hover {
+      color: $button-color;
+      border: 1px solid $button-color;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover